### PR TITLE
resolves #2710 by stopping start.js from overwriting user.js settings

### LINKED ISSUE #2710: Editor settings not persisting on local version
I have a local version of JsBin running and everything works as expected except the editor settings aren't persisting correctly. I have inspected the sqlite database and the changes are being saved there, but they aren't being applied to the editor and have also reverted back to the defaults in the settings window.

To replicate:
- Login
- Open Account > Editor settings
- Change something (reports that settings are saved)
- Back to JSBin
- Open Account > Editor settings
- Changes aren't persisted

### PR DESCRIPTION
**Pull request by jtfell**
Allows editor settings to persist when running locally. Not sure if this will affect users logged in via Github

### CODE REVIEW

File: views/user.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-var jsbin = {};
```
Comment (remy) [inline views/user.html]: Does `start.js` define `jsbin`? since it's now being removed from this line...

**Comment by jtfell** [comment]
Yes it does, that was the issue. start.js was overwriting the user.js version of jsbin.

**Comment by remy** [comment]
I'd like to check this in both production mode and non-production mode.

**Comment by jtfell** [comment]
No worries, do whatever you like with it. It's working for me locally so I thought you might be interested. Thanks for the advice earlier!

**Comment by jtfell** [comment]
Any progress on this? I'm curious if it works in production.

**Pull request closed without merging**